project/TestPhases.scala

```diff
@@ -0,0 +1,10 @@
+import sbt.{ForkOptions, TestDefinition}
+import sbt.Tests.{Group, SubProcess}
+
+object TestPhases {
+
+  def oneForkedJvmPerTest2(tests: Seq[TestDefinition]) =
```

Comment:
why is there a "2" in the name ?

Reply:
also currently there is only one test phase, move this function into build.sbt and make it private

Reply:
At one time, at least, there was an existing oneForkedJvmPerTest which we could not override.  If it works, then there is no reason why it should not be oneForkedJvmPerTest now